RevenueCat/Scripts/Offering.cs

```diff
@@ -0,0 +1,58 @@
+using System.Collections.Generic;
+using JetBrains.Annotations;
+
+public partial class Purchases
+{
+    public class Offering
+    {
+        public readonly string Identifier;
+        public readonly string ServerDescription;
+        public readonly List<Package> AvailablePackages;
+        [CanBeNull] public readonly Package Lifetime;
+        [CanBeNull] public readonly Package Annual;
+        [CanBeNull] public readonly Package SixMonth;
+        [CanBeNull] public readonly Package ThreeMonth;
+        [CanBeNull] public readonly Package TwoMonth;
+        [CanBeNull] public readonly Package Monthly;
+        [CanBeNull] public readonly Package Weekly;
```

Comment:
or is the criteria: 
- `[CanBeNull]` + `Package?` if it can validly be _assigned_ to null
- `[CanBeNull]` + `Package` if it can only be null if the value has never been set